x-pack/platform/plugins/shared/ml/public/application/timeseriesexplorer/components/series_controls/series_controls.tsx

```diff
@@ -321,7 +321,7 @@ export const SeriesControls: FC<PropsWithChildren<SeriesControlsProps>> = ({
 
   return (
     <div data-test-subj="mlSingleMetricViewerSeriesControls">
-      <EuiFlexGroup direction={direction}>
+      <EuiFlexGroup direction={direction} gutterSize="s">
```

Comment:
Can the detector dropdown be given a minimum width? Also is there any easy way to preserve the vertical alignment of the model bounds, annotations and forecast controls as the width is reduced?

<img width="851" alt="Screenshot 2024-12-17 at 17 07 29" src="https://github.com/user-attachments/assets/3614a23a-72ae-4fab-a582-771ecd63001f" />

Reply:
Updated the detector dropdown to have a min width in: [#cd56f03](https://github.com/elastic/kibana/pull/203813/commits/cd56f0388aa672590b349fbd015eef4b1594dc1e)

To preserve vertical alignment, I propose wrapping controls as the width decreases. I don't see any way to keep everything in a single row while maintaining good alignment.

WDYT?

https://github.com/user-attachments/assets/f80a4507-f785-4ad3-a96c-abc1c667f75f

Reply:
This looks good, I think it is a good solution.

Reply:
Agree with @joana-cps - this is working well.